libs/net/dal/Services/UserService.cs

```diff
@@ -50,7 +50,7 @@ public IPaged<User> Find(UserFilter filter)
             $"{keyword}%") || EF.Functions.Like(c.LastName.ToLower(), $"{keyword}%"));
         }
         if (!String.IsNullOrWhiteSpace(filter.RoleName))
-            query = query.Where(c => c.Roles.Any(r => r.Name.ToLower() == filter.RoleName.ToLower()));
+            query = query.Where(c => EF.Functions.Like(c.Roles.ToLower(), $"%[{filter.RoleName.ToLower()}]%"));
```

Comment:
I don't love this solution for searching, but I couldn't think of simple way to search two different data sources (keycloak, tno), and provide paging.